service/switchboardapp/src/test/java/com/switchboard/app/fixture/EncoderFixture.java

```diff
@@ -0,0 +1,26 @@
+package com.switchboard.app.fixture;
+
+import com.switchboard.app.domain.DeviceEntity;
+import com.switchboard.app.domain.EncoderEntity;
```

Comment:
`import com.switchboard.app.domain.DeviceEntity;` needs to change to `import com.switchboard.app.entity.DeviceEntity;`

`import com.switchboard.app.domain.EncoderEntity;` needs to change to `import com.switchboard.app.entity.EncoderEntity;`

If not changed, we will get a compile error on merge.

Reply:
Has been updated